### ISSUE TITLE
Error in training code

### ISSUE THREAD

**Issue by Mukil07** [zmehdiz97/bddoia_project#1]
I am trying to reproduce their work, I pretrained the backbone FRCNN on BDD100k it gave me 29AP. Now when i used that weights to train bddoia model. All the logits remain the same and the loss function saturates to 0.77. Could you please help me figure out?

Also, did you make any changes to the train.py file ? 

Thanks !!
